vhost-user-backend/Cargo.toml

```diff
@@ -14,14 +14,14 @@ xen = ["vm-memory/xen", "vhost/xen"]
 [dependencies]
 libc = "0.2.39"
 log = "0.4.17"
-vhost = { path = "../vhost", version = "0.9", features = ["vhost-user-backend"] }
+vhost = { path = "../vhost", version = "0.10", features = ["vhost-user-backend"] }
```

Comment:
That should be moved to the next commit, right?

Reply:
You're right, where did the vmm-sys-util update commit go? :thinking: 
I must've messed up a rebase somewhere :(

Reply:
Ahh wait, vmm-sys-util got merged yesterday, I just messed up resolving the conflict when I re-ordered the vm-memory commit to the start of this series. Fixed now!